linter_exclusions.yml

```
@@ -319,6 +319,8 @@ codespace plan create:
     default_sku_name:
       rule_exclusions:
       - option_length_too_long
+confidentialledger managedccfs update:
+- missing_command_test_coverage
```

```suggestion
confidentialledger managedccfs update:
  rule_exclusions:
  - missing_command_test_coverage
```
After I reconsidered, it should be modified like that.

---

Fixed